doc/user-changes.md

```diff
@@ -4,6 +4,24 @@ This document is a development diary summarizing changes in `alr` that notably
 affect the user experience. It is intended as a one-stop point for users to
 stay on top of `alr` new features.
 
+### New `alr with --versions` switch
+
+PR [#464](https://github.com/alire-project/alire/pull/464).
+
+A new `alr with --versions` switch is available to obtain version-focused
+information of dependencies. Namely, the combined dependencies on a crate are
+shown, with the release in the solution, and the last known version for the
+crate:
+```
+CRATE      DEPENDENCY      SOLVED  LATEST 
+a_project  (root)          0.0.0   unknown
```

Comment:
Why the latest version is not `0.0.0` here?

Reply:
Because `unknown` is used whenever there is no release in the index. 

In this case it may make sense to use the local version of the crate, although I think I prefer `unindexed` to make clear that there's no index information (see the other comment) for the crate.